enctests/testframework/encoders/ffmpeg_encoder.py

```diff
@@ -81,7 +81,16 @@ def run_wedges(self) -> dict:
             result_meta['completed_utc'] = \
                 datetime.now(timezone.utc).isoformat()
             result_meta['encode_time'] = round(enctime, 4)
-            result_meta['filesize'] = out_file.stat().st_size
+            if os.path.exists(out_file) and out_file.stat().st_size > 0:
+                result_meta['filesize'] = out_file.stat().st_size
+            else:
+                print(f"ERROR: {test_name} failed to create a file {out_file}")
+                print(f"\tERROR: {test_name} failed to create a file ")
+                log_file_object.close()
+                with open(log_file, "r") as log_file_object:
+                    print("Output from file generation:\n", log_file_object.read())
+                result_meta['filesize'] = -1
+                
```

Comment:
```suggestion
            try:
                result_meta['filesize'] = out_file.stat().st_size
                
            except FileNotFoundError:
                print("ERROR: {test_name} failed to create a file {out_file}")
                print(f"\tERROR: {test_name} failed to create a file ")
                log_file_object.close()
                with open(log_file, "r") as log_file_object:
                    print("Output from file generation:\n", log_file_object.read())
                result_meta['filesize'] = -1
                
```
Simplified a bit so we don't need to do so many tests.
Alternatively just use `out_file.exists() and out_file.stat().st_size > 0` in the tests to utilize the Path object

Reply:
FFmpeg will create an empty file in some cases, so we do need to check for both and treat an empty file as an error too.

Reply:
OK, I see. In that case my only suggestion is: `out_file.exists() and out_file.stat().st_size > 0`